lib/pure/parseopt.nim

```diff
@@ -95,7 +95,11 @@ when declared(os.paramCount):
   proc initOptParser*(cmdline = "", shortNoVal: set[char]={},
                       longNoVal: seq[string] = @[]): OptParser =
     ## inits the option parser. If ``cmdline == ""``, the real command line
-    ## (as provided by the ``OS`` module) is taken.
+    ## (as provided by the ``OS`` module) is taken.  If ``shortNoVal`` is
+    ## provided command users need not delimit short option keys and values
+    ## with a ':' or '='.  If ``longNoVal`` is provided command users need not
```

Comment:
`is provided command users need not` maybe this can be changed? :page_facing_up: